style.css

```diff
@@ -0,0 +1,104 @@
+body,
+h1,
+h2,
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  padding-left: 24px;
+  padding-right: 24px;
+  display: flex;
+}
+
+nav {
+  width: 98%;
+  height: 60px;
+  background: #3c3a39;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 10px;
+}

```

Comment:
- [ ] Kindly remove `width: 98%;` and `padding: 10px;` from here, to solve the white space on the left issue, as required by the previous reviewer.

![image](https://user-images.githubusercontent.com/35535130/158914857-e0045181-663e-413d-a966-cc4d2142ea22.png)